lib/engine.py

```diff
@@ -132,6 +133,7 @@ def source_gen():
                                    for n in range(len(linevalues))]
                 except Exception as e:
                     self.warning('Exception in line %s: %s' % (self.table.record_id, e))
+                    exit()
```

Comment:
Oh that was left by mistake. It was used for debugging.
I did not see that as I was checking the diff.